ui/src/app/app.component.spec.ts

```diff
@@ -52,6 +57,9 @@ describe('AppComponent', () => {
     }).compileComponents();
 
     mockStore = TestBed.inject(MockStore);
+    mockRouter = TestBed.inject(Router);
+    fixture = TestBed.createComponent(AppComponent);
```

Comment:
Sonar: Variables should not be shadowed
`fixture` is already used in some test cases and shadow this global `fixture`

Reply:
Fixed